src/components/Article.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Article({ title, date, text }) {
+  return (
+    <>
+      <h1>
+        {title}
+      </h1>
+      <span>
+        {date}
+      </span>
+      <p>
+        {text}
+      </p>
+    </>
```

Comment:
You don't need to use fragment (<>), it's should be an article.
Where are class names? 
Try to use the arrow function, I think it's more clearly

Reply:
Thanks, link is live now.

Neither `<article>`, nor classes are required according to the task.